<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>direction_sprite_widget</class>
 <widget class="QWidget" name="direction_sprite_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_main">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="layout_sprite_info">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gb_sprite">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>110</height>
         </size>
        </property>
        <property name="title">
         <string>Sprite Information</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>191</width>
           <height>74</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_name">
            <property name="text">
             <string>Name :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_name"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_width">
            <property name="text">
             <string>Width :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_width">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_height">
            <property name="text">
             <string>Height :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="le_height">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_preview">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Preview</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>121</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="layout_preview">
          <item>
           <widget class="QLabel" name="lbl_preview">
            <property name="text">
             <string>P</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sp_fps">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>15</number>
            </property>
            <property name="maximum">
             <number>150</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_frame">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Add a frame...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_frames">
        <property name="text">
         <string>Frame(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="1">
         <widget class="QComboBox" name="cb_direction"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_direction">
          <property name="text">
           <string>Direction :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="sa_frames">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>549</width>
         <height>289</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
